src/templates/en/2019/base.html

```diff
@@ -73,7 +75,9 @@ <h2>
   </header>
   {% endblock %}
 
-  {% block main %}{% endblock %}
+  <div class="container">
+    {% block main %}{% endblock %}
+  </div>
 
   {% block footer %}
     <footer class="alt-bg">
```

Comment:
Why is the footer not in a container div? Currently it's very wide compared to everything else on the page.

Reply:
I decided to let the footer that wide on purpose, as I thought it looks pretty good to me - from a design perspective - considering its current content: logo, menu, social, icons, copyright info.

Also, having a full-width footer is most of the time a common practice in web design/development.

Reply:
Yeah seen it before but not a fan of that design principal. Especially when not a huge amount in our footer to need that. But hey that's a personal opinion and sure there's plenty of others who prefer this design.

The [original design](https://github.com/HTTPArchive/almanac.httparchive.org/issues/46#issuecomment-515966879) has narrower footer, then again it has wider header - that we are changing with this PR - so not sure that's helping my argument! 😀

Happy to resolve this with no change if you think it's working as intended.

Reply:
TBH lots of things are a bit different compared to the original design, and that's not necessarily a bad thing, because this is what it happens usually when you implement a design :)

Yes, in my opinion, the footer looks and works as intended. Thanks!